### PR TITLE
[WEBSITE] Press backend model

### PR DESCRIPTION
**Pull request by FaithDaka**
#### Summary of Changes (What does this PR do?)

- Features the Press page backend
- Publications model also enhanced to include an attribute for resource files

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] The title of the PR states what changed and the related issues number (used for the release note).
- [x] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

#### How should this be manually tested?

- Using deploy preview, sign into the admin portal to interact with the press table. Add, edit and delete entries to test.

#### What are the relevant tickets?

- [WEB-403]

#### Screenshots
image
image

[WEB-403]: https://airqoteam.atlassian.net/browse/WEB-403?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: website/backend/press/models.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+@with_author
+class Press(BaseModel):
+    article_title = models.CharField(max_length=100)
+    article_intro = models.CharField(max_length=200, null=True, blank=True)
+    article_link = models.URLField(null=True, blank=True)
+    date_published = models.DateField()
+    publisher_logo = CloudinaryField("PublisherLogo", overwrite=True, resource_type="image", null=True, blank=True)
+    order = models.IntegerField(default=1)
+
```
Comment (FaithDaka) [inline website/backend/press/models.py:14]: @OchiengPaul442 Let me know if you'll need any other attributes on the frontend

Comment (OchiengPaul442) [reply website/backend/press/models.py:14]: I will. thanks, @FaithDaka

**Review by Baalmart — APPROVED** [review]
Looks good to me, thanks @FaithDaka

File: website/backend/press/models.py (hunk 1 above)
Comment (Baalmart) [reply website/backend/press/models.py:14]: Hi @FaithDaka , just a followup on the Cloudinary file orgnaisation. Are we storing these files in a "nice way"? Say like in some hierarchical folder structure of sorts for easy management?

**Pull request merged**